services/blockchain-connector/shared/sdk/peerCache.js

```diff
@@ -45,7 +46,7 @@ const refactorPeer = (orgPeer, state) => {
 	peer.state = state;
 	peer.height = height;
 	peer.ip = ipAddress;
-	return peer;
+	return parseToJSONCompatObj(peer);
```

Comment:
Seems like an issue from the SDK. For now, please update the method as below:

```
const {
	Logger,
	Signals,
	Utils: { isObject },
} = require('lisk-service-framework');

const refactorPeer = (orgPeer, state) => {
	const { chainID, ipAddress, options: { height } = {}, ...peer } = orgPeer;
	peer.chainID = isObject(chainID)
		? Buffer.from(chainID.data).toString('hex')
		: chainID;
	peer.state = state;
	peer.height = height;
	peer.ip = ipAddress;
	return peer;
};
```

Reply:
Created issue in the SDK repo to fix the same: https://github.com/LiskHQ/lisk-sdk/issues/8173.
Please add a TODO to revert this change, once SDK addressed the above issue.